src/olympia/shelves/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from olympia.amo.tests import TestCase, reverse_ns
+from olympia.shelves.models import Shelf, ShelfManagement
+
+
+class TestShelfViewSet(TestCase):
+    def setUp(self):
+        self.url = reverse_ns('shelves-list', api_version='v4')
```

Comment:
do you need to specify the `api_version` here?  (it should default to 'v5' which should work okay too)

Reply:
I'll remove `api_version` as it did work without it. I had made the endpoint for the shelves API `/api/v4/shelves/`, should it be changed?

Reply:
> I had made the endpoint for the shelves API /api/v4/shelves/, should it be changed?

where did you do that?

Reply:
Please see below.